pkg/api/v1/externalreference.go

```diff
@@ -0,0 +1,8 @@
+package v1
+
+type ExternalProjectReference struct {
+	// ID is the Atlas project ID
+	ID string `json:"id"`
```

Comment:
is this required? if so, please add a kubebuilder annotation.

Reply:
I will take this off from @helderjs and will add the kubebuilder annotations here

Reply:
This is back now on @helderjs hands